lib/route.js

```diff
@@ -0,0 +1,18 @@
+SharedRoute = class {
+  constructor(router, pathDef, options) {
+    options = options || {};
+    
+    this.options = options;
+
+    this.name = options.name;
+
+    this.pathDef = pathDef;
+  
+    // Route.path is deprecated and will be removed in 3.0
+    this.path = this.pathDef;
+    
+    this._router = router;
```

Comment:
Are you sure you want to remove `_` from protected methods? From a quick glance online it seems like both private/protected members/methods have a `_` before them.